Panic when trying to login to GitHub

### Describe the bug

I get an "index out of range" error when trying to login to GitHub using HTTPS protocol.
I'm running on a M1 MacBook, and installed gh cli using Homebrew.

gh version 2.14.5 (2022-08-23)
https://github.com/cli/cli/releases/tag/v2.14.5

### Steps to reproduce the behavior

1. Type 'gh auth login'
2. Select GitHub.com
3. Select HTTPS
4. See the error: `panic: runtime error: index out of range [-1]`

### Expected vs actual behavior

Under normal circumstances I'd expect to get a link to authenticate on a web browser.

### Logs

```
% gh auth login 
? What account do you want to log into? 
? What is your preferred protocol for Git operations? 
panic: runtime error: index out of range [-1]

goroutine 1 [running]:
github.com/cli/cli/v2/pkg/cmd/auth/shared.Login(0x1400073f9f8)
        github.com/cli/cli/v2/pkg/cmd/auth/shared/login_flow.go:60 +0x1078
github.com/cli/cli/v2/pkg/cmd/auth/login.loginRun(0x140001b8630)
        github.com/cli/cli/v2/pkg/cmd/auth/login/login.go:175 +0x390
github.com/cli/cli/v2/pkg/cmd/auth/login.NewCmdLogin.func1(0x14000306f00?, {0x1020d9e08?, 0x0?, 0x0?})
        github.com/cli/cli/v2/pkg/cmd/auth/login/login.go:111 +0x490
github.com/spf13/cobra.(*Command).execute(0x14000306f00, {0x1020d9e08, 0x0, 0x0})
        github.com/spf13/cobra@v1.4.0/command.go:856 +0x4c4
github.com/spf13/cobra.(*Command).ExecuteC(0x140001c6c80)
        github.com/spf13/cobra@v1.4.0/command.go:974 +0x354
main.mainRun()
        github.com/cli/cli/v2/cmd/gh/main.go:214 +0xaa0
main.main()
        github.com/cli/cli/v2/cmd/gh/main.go:48 +0x1c
```

--- a/pkg/cmd/auth/login/login.go
+++ b/pkg/cmd/auth/login/login.go
@@ -187,13 +187,11 @@ func loginRun(opts *LoginOptions) error {
 }
 
 func promptForHostname(opts *LoginOptions) (string, error) {
+	options := []string{"GitHub.com", "GitHub Enterprise Server"}
 	hostType, err := opts.Prompter.Select(
 		"What account do you want to log into?",
-		"",
-		[]string{
-			"GitHub.com",
-			"GitHub Enterprise Server",
-		})
+		options[0],
+		options)
 	if err != nil {
 		return "", err
 	}

--- a/pkg/cmd/auth/shared/login_flow.go
+++ b/pkg/cmd/auth/shared/login_flow.go
@@ -53,7 +53,8 @@ func Login(opts *LoginOptions) error {
 		}
 		result, err := opts.Prompter.Select(
 			"What is your preferred protocol for Git operations?",
-			"", options)
+			options[0],
+			options)
 		if err != nil {
 			return err
 		}
@@ -82,7 +83,8 @@ func Login(opts *LoginOptions) error {
 		if len(pubKeys) > 0 {
 			options := append(pubKeys, "Skip")
 			keyChoice, err := opts.Prompter.Select(
-				"Upload your SSH public key to your GitHub account?", "",
+				"Upload your SSH public key to your GitHub account?",
+				options[0],
 				options)
 			if err != nil {
 				return err
@@ -126,12 +128,12 @@ func Login(opts *LoginOptions) error {
 	if opts.Web {
 		authMode = 0
 	} else if opts.Interactive {
+		options := []string{"Login with a web browser", "Paste an authentication token"}
 		var err error
 		authMode, err = opts.Prompter.Select(
-			"How would you like to authenticate GitHub CLI?", "",
-			[]string{
-				"Login with a web browser",
-				"Paste an authentication token"})
+			"How would you like to authenticate GitHub CLI?",
+			options[0],
+			options)
 		if err != nil {
 			return err
 		}

--- a/pkg/cmd/extension/command.go
+++ b/pkg/cmd/extension/command.go
@@ -247,11 +247,10 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 				if err != nil {
 					return extName, -1, err
 				}
-				extTmplType, err := prompter.Select("What kind of extension?", "", []string{
-					"Script (Bash, Ruby, Python, etc)",
-					"Go",
-					"Other Precompiled (C++, Rust, etc)",
-				})
+				options := []string{"Script (Bash, Ruby, Python, etc)", "Go", "Other Precompiled (C++, Rust, etc)"}
+				extTmplType, err := prompter.Select("What kind of extension?",
+					options[0],
+					options)
 				return extName, extensions.ExtTemplateType(extTmplType), err
 			}
 			var flagType string

--- a/pkg/cmd/pr/review/review.go
+++ b/pkg/cmd/pr/review/review.go
@@ -205,9 +205,11 @@ func reviewRun(opts *ReviewOptions) error {
 }
 
 func reviewSurvey(opts *ReviewOptions, editorCommand string) (*api.PullRequestReviewInput, error) {
+	options := []string{"Comment", "Approve", "Request Changes"}
 	reviewType, err := opts.Prompter.Select(
-		"What kind of review do you want to give?", "",
-		[]string{"Comment", "Approve", "Request Changes"})
+		"What kind of review do you want to give?",
+		options[0],
+		options)
 	if err != nil {
 		return nil, err
 	}

